build/build-in-docker

```diff
@@ -24,30 +24,29 @@ set -e
 SCRIPTDIR=$(cd $(dirname $0); pwd)
 
 LOCAL_MAVEN_REPO=${LOCAL_MAVEN_REPO:-"$HOME/.m2/repository"}
-CUDF_USE_PER_THREAD_DEFAULT_STREAM=${CUDF_USE_PER_THREAD_DEFAULT_STREAM:-ON}
 USE_GDS=${USE_GDS:-ON}
 export CMAKE_GENERATOR=${CMAKE_GENERATOR:-"Ninja"}
+export DOCKER_BUILD_EXTRA_ARGS="--build-arg TARGETPLATFORM=linux/amd64 --build-arg CMAKE_ARCH=x86_64"
+CUDA_VERSION=${CUDA_VERSION:-cuda11}
```

Comment:
Argh, run-in-docker expects CUDA_VERSION to be 11.8.0 not cuda11, but this script needs the classifier version.  To make it consistent, I think it needs to look like this:
```suggestion
export CUDA_VERSION=${CUDA_VERSION:-11.8.0}
CUDA_CLASSIFIER=cuda${CUDA_VERSION%%.*}
```
and use $CUDA_CLASSIFIER where CUDA_VERSION is used below.

Reply:
Good catch, fixed